charts/vector/templates/secret-config.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if not .Values.existingConfigMaps }}
```

Comment:
The file name can be changed from `charts/vector/templates/secret-config.yaml` to `charts/vector/templates/vector-config.yaml`